src/fu-util.c

```diff
@@ -1514,6 +1499,10 @@ fu_util_update_device_with_release (FuUtilPrivate *priv,
 				    FwupdRelease *rel,
 				    GError **error)
 {
+	if (!priv->assume_yes) {
```

Comment:
This needs another check for interactive.  Otherwise I expect this breaks people who script `fwupdmgr update`

Reply:
Maybe we just use `no_safety_check`?